locales/hi_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -645,7 +644,7 @@ msgid "Chat"
 msgstr "बातचीत "
 
 msgid "Check %sMake this the current search engine%s"
-msgstr "%s %s"
+msgstr "%sइसे अपना खोज यन्त्र बनाएँ%s"
```

Comment:
Check translation missing. This is in the context to tick mark on `Make this the current search engine`.